src/Configuration.php

```diff
@@ -66,6 +69,7 @@ public function __construct(
         $this->setEventDispatcher($eventDispatcher);
         $this->setBoundary(sha1(microtime()));
         $this->setUserAgent('Ivory Http Adapter '.HttpAdapterInterface::VERSION);
+        $this->setSSLVerifyPeer(true);
```

Comment:
Is it the default option for all adapters? If no, can you track the changes in the UPGRADE file?